source/_integrations/modbus.markdown

```diff
@@ -454,6 +466,54 @@ climates:
       required: false
       type: string
       default: C
+    hvac_mode_register:
+      description: Definition of a register holding and controlling an HVAC mode
+      required: false
+      type: [map]
+      keys:
+        address: 
+          description: The address of the HVAC mode register.
+          required: true
+          type: integer
+        values:
+          description: A mapping between the register values and HVAC modes
+          required: true
```

Comment:
Why is this required, when all the entries above are optional ?

Reply:
If an HVAC mode register is defined, at least one value is optional here, but each individual one is optional.